lib/simulator.py

```diff
@@ -241,11 +248,16 @@ def on_size(self, e):
         setsize_window_width = self.width * self.scale + decorations_width + self.margin * 2
         setsize_window_height = (self.height) * self.scale + decorations_height + self.margin * 2
 
+        # set minimum width (force space for control buttons)
         if setsize_window_width < 600:
             setsize_window_width = 600
 
         self.SetSize(( setsize_window_width, setsize_window_height))
 
+        # center the simulation on screen if not called by params
+        if not wx.Window.FindWindowById(9876):
```

Comment:
Instead of looking to see if the settings window exists, perhaps we can have Params tell `EmbroiderySimulator` not to center itself.  How about adding a `no_center` argument to `EmbroiderySimulator`?  The default would be to center, but if the code that instantiates `EmbroiderySimulator` (params in this case) asks not to center, then don't.